cmd/chisel/cmd_find.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/canonical/chisel/internal/setup"
+	"github.com/canonical/chisel/internal/strdist"
+)
+
+var shortFindHelp = "Find existing slices"
+var longFindHelp = `
+The find command queries the slice definitions for matching slices.
+Globs (* and ?) are allowed in the query.
+
+By default it fetches the slices for the latest Ubuntu
+version, unless the --release flag is used.
```

Comment:
Same as above.

Reply:
You are completely right, changed in https://github.com/canonical/chisel/pull/99/commits/d124e2503566c90d7702f9db1f42ac7b938cb283.